front/src/layout/AppLayout.tsx

```diff
@@ -28,12 +26,10 @@ type OwnProps = {
 
 function AppLayout({ children, user }: OwnProps) {
   return (
-    <ThemeProvider theme={lightTheme}>
```

Comment:
I think we need to keep ThemeProvider at AppLayout level as we will need to get it from the user at long term, we cannot do that from index.ts

Reply:
Yes I tried doing so initially but then it was harder to have the entire App work well in storybook. People/Companies pages worked well but if you put the theme in Layout then the storybook toggle will be ignored. I can give it another shot